test/Chainweb/Test/Pact/TransactionTests.hs

```diff
@@ -372,5 +373,5 @@ logResults = fmap f
     f l =
       ( _txDomain l
       , _txKey l
-      , l ^? txValue . _Object . ix "balance"
+      , l ^? txValue . _Object . ix "balance" <|> l ^? txValue . _Object . ix "$d" . _Object . ix "balance"
```

Comment:
I don't understand this change so it probably needs an explanatory comment for maintainers.